osu.Game.Rulesets.Catch/Resources/Testing/Beatmaps/basic-expected-conversion.json

```diff
@@ -0,0 +1 @@
271.0,"Position":481.0},{"StartTime":19331.0,"Position":92.0},{"StartTime":19390.0,"Position":211.0},{"StartTime":19450.0,"Position":135.0}]},{"StartTime":19875.0,"Objects":[{"StartTime":19875.0,"Position":216.0},{"StartTime":19937.0,"Position":215.307053},{"StartTime":20000.0,"Position":236.036865},{"StartTime":20062.0,"Position":236.312088},{"StartTime":20125.0,"Position":235.838928},{"StartTime":20187.0,"Position":269.9743},{"StartTime":20250.0,"Position":285.999146},{"StartTime":20312.0,"Position":283.669067},{"StartTime":20375.0,"Position":317.446747},{"StartTime":20437.0,"Position":330.750275},{"StartTime":20500.0,"Position":344.0156},{"StartTime":20562.0,"Position":318.472168},{"StartTime":20625.0,"Position":309.165466},{"StartTime":20687.0,"Position":317.044617},{"StartTime":20750.0,"Position":280.457367},{"StartTime":20812.0,"Position":272.220581},{"StartTime":20875.0,"Position":270.3294},{"StartTime":20937.0,"Position":262.57605},{"StartTime":21000.0,"Position":244.803329},{"StartTime":21062.0,"Position":215.958359},{"StartTime":21125.0,"Position":177.79332},{"StartTime":21187.0,"Position":190.948349},{"StartTime":21250.0,"Position":158.78334},{"StartTime":21312.0,"Position":136.93837},{"StartTime":21375.0,"Position":119.121056},{"StartTime":21437.0,"Position":132.387573},{"StartTime":21500.0,"Position":124.503014},{"StartTime":21562.0,"Position":118.749374},{"StartTime":21625.0,"Position":123.165535},{"StartTime":21687.0,"Position":96.02999},{"StartTime":21750.0,"Position":118.547928},{"StartTime":21812.0,"Position":128.856232},{"StartTime":21875.0,"Position":124.28746},{"StartTime":21937.0,"Position":150.754929},{"StartTime":22000.0,"Position":149.528732},{"StartTime":22062.0,"Position":145.1691},{"StartTime":22125.0,"Position":182.802155},{"StartTime":22187.0,"Position":178.6452},{"StartTime":22250.0,"Position":213.892181},{"StartTime":22312.0,"Position":218.713028},{"StartTime":22375.0,"Position":240.4715},{"StartTime":22437.0,"Position":239.371887},{"StartTime":22500.0,"Position":261.907257},{"StartTime":22562.0,"Position":314.353119},{"StartTime":22625.0,"Position":299.273376},{"StartTime":22687.0,"Position":356.98288},{"StartTime":22750.0,"Position":339.078552},{"StartTime":22812.0,"Position":377.8958},{"StartTime":22875.0,"Position":398.054047},{"StartTime":22937.0,"Position":398.739441},{"StartTime":23000.0,"Position":407.178467},{"StartTime":23062.0,"Position":444.8687},{"StartTime":23125.0,"Position":417.069977},{"StartTime":23187.0,"Position":454.688477},{"StartTime":23250.0,"Position":428.9612},{"StartTime":23312.0,"Position":441.92807},{"StartTime":23375.0,"Position":439.749878},{"StartTime":23433.0,"Position":455.644684},{"StartTime":23491.0,"Position":440.7359},{"StartTime":23549.0,"Position":430.0944},{"StartTime":23607.0,"Position":420.796173},{"StartTime":23665.0,"Position":435.897461},{"StartTime":23723.0,"Position":418.462555},{"StartTime":23781.0,"Position":405.53775},{"StartTime":23874.0,"Position":408.720825}]}]}
```

Comment:
let's prettify this so it will diff well if we make changes

Reply:
Not that we should ever be making manual changes to this file, and it's autogenerated.

Reply:
it's exactly for that scenario (so when automatic change are made we can see the diff easily).